### PR TITLE
Validate file uploads

### PR DESCRIPTION
**Pull request by ajkiessl**
Validate that format review file and final submission file is uploaded at corresponding stages.  Test for this.

### CODE REVIEW

File: app/models/submission.rb [hunk 1]
```diff
@@ -405,17 +405,22 @@ def proquest_agreement=(input)
...
-      else
-        true
+
+      if status_behavior.collecting_format_review_files?
+        return true unless format_review_files.nil? || format_review_files.blank?
+
+        errors.add(:format_review_file, "You must upload a Format Review file.")
+
+      elsif status_behavior.collecting_final_submission_files?
+        return true unless final_submission_files.nil? || final_submission_files.blank?
```
Comment (Smullz622) [inline app/models/submission.rb:418]: Doesn't blank include nil? Is the OR needed here & above?

**Review by Smullz622 — APPROVED** [review]
have a quick question, but otherwise lgtm!

File: app/models/submission.rb (hunk 1 above)
Comment (ajkiessl) [reply app/models/submission.rb:418]: Oh right, yes.  `blank?` should be enough here.

**Review by Smullz622 — APPROVED** [review]
lgtm!

**Pull request merged**